github/github.go

```diff
@@ -144,6 +147,11 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// BreakdownOptions specifies the parameters to methods that support breakdown per day or week.
+type BreakdownOptions struct {
+	Per string `url:"per,omitempty"`
```

Comment:
I think it'll be very helpful to document this `Per` parameter with something like this:

```
// Can be one of: day, week. Default: day.
```

For example, see https://github.com/google/go-github/blob/b9698164669e386680621e2006cab02271257611/github/repos.go#L113-L115.